src/index.html

```diff
@@ -20,25 +22,63 @@ <h2 class="hero-subheader small-10 medium-10 large-10 small-offset-1 medium-offs
-                        <div class="row column text-center"><a href="/vae/hub/oauth_login" class="large button sites-button">Login with GitHub</a></div>
+                    <div>
+                      <h1 class="text-center">Public access</h1>
+                      <p>
+                        We have a set of Jupyter Notebooks that can be run by anyone without a login.
+                      </p>
+                      <div class="row column text-center">
+                        <a href="/public/hub/tmplogin" class="large button sites-button">Click here to obtain a temporary session</a>
+                      </div>
+                    </div>
+
+                    <hr class="whitespace">
+
+                    <div>
+                      <h1 class="text-center">GitHub login</h1>
+                      <p>
+                        If you are interested in developing your own data analysis notebooks on our analysis platform please contact us with your GitHub username.
+                        This will give you more computational resources including scratch storage space than is available through the public access.
+                      </p>
+                      <p>
+                        We will do our best to maintain this scratch space across upgrades of the VAE, though in the event of unforeseen issues we may need to delete all scratch space, and you should consider it as suitable for temporary storage only.
+                        If as part of a planned update we need to delete our existing storage we will give at least one week's notice on <a href="https://github.com/idr-contrib/community/issues"> idr-contrib/community</a>.
+                      </p>
+                      <div class="row column text-center">
+                        <a href="/vae/hub/oauth_login" class="large button sites-button">Login with GitHub</a>
+                      </div>
+                    </div>
 
-                    <h2>Elixir AAI login</h2>
-                        <p>If you have an Elixir AAI account you can use it to access the analysis platform. This is currently a demonstrator, and is identical to the public access. Please note we are unable to investigate authentication errors between your chosen identity provider and Elixir AAI.</p>
-                        <div class="row column text-center"><a href="/vae-aai/hub/oauth_login" class="large button sites-button">Login with Elixir  AAI</a></div>
+                    <hr class="whitespace">
 
-                    <h2>More information</h2>
-                        <p>For more information about the IDR please see the <a href="https://idr.openmicroscopy.org/">IDR home page</a>.</p>
+                    <div>
+                      <h1 class="text-center">Elixir AAI login</h1>
```

Comment:
I keep going back and forth on whether to have `(experimental)` in the title of this one...